apps/crn-server/src/data-providers/reminders.data-provider.ts

```diff
@@ -346,6 +364,91 @@ const getResearchOutputDraftRemindersFromQuery = (
   );
 };
 
+const getResearchOutputInReviewRemindersFromQuery = (
+  queryResearchOutputsContents: FetchReminderDataQuery['inReviewResearchOutputs'],
+  findUsersContent: FetchReminderDataQuery['findUsersContent'],
+): ResearchOutputInReviewReminder[] => {
+  if (!findUsersContent || !findUsersContent.flatData.teams) {
+    return [];
+  }
+
+  const userTeamIds = getUserTeamIds(findUsersContent.flatData.teams);
+  const userWorkingGroupIds = getUserWorkingGroupIds(
+    findUsersContent.referencingWorkingGroupsContents,
+  );
+
+  const isAsapStaff = findUsersContent.flatData.role === 'Staff';
```

Comment:
This condition is not quite right, from the ticket it seems that we should show this reminder to PM's as well. Also, it's possible to use the function `isCMSAdministrator` to check if the user is a staff

> The ready for review reminder should be displayed to PMs/Admins until the output has been published or switched back to draft

Reply:
There are two roles we evaluate here. The ASAP role that can be `'Staff', 'Grantee', 'Guest', 'Hidden'`
![Screenshot 2023-06-28 at 06 19 55](https://github.com/yldio/asap-hub/assets/16595804/89e1c7f4-5ea5-4f2a-9d3b-8e0acfcc3dfa)

and the team role
![Screenshot 2023-06-28 at 06 18 58](https://github.com/yldio/asap-hub/assets/16595804/afa6b6fb-dd01-422f-a648-8cded26ff260)
that can be 

```
  'Lead PI (Core Leadership)',
  'Co-PI (Core Leadership)',
  'Collaborating PI',
  'Project Manager',
  'Key Personnel',
  'Scientific Advisory Board',
  'ASAP Staff',
```

you're only checking the ASAP role now. But a user could be a `Grantee` and a `Project Manager` and they wouldn't see the reminder.